modules/argo-cd-command-line.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * argo-cd-custom-resource-properties.adoc
+
+:_content-type: PROCEDURE
+[id="command-line-tool_{context}"]
+= Command line tool
+
+[role="_abstract"]
+The command line tool for Argo CD is not supported.
```

Comment:
This sentence is not clear to me in two aspects:

1. `The command line tool for Argo CD is not supported` [where? on OpenShift?]`.`

2. `The command line tool for Argo CD` is a generic definition. The questions that this raises in my head are where can I find more information about it and, if it has a name, then how is this tool called?
 
In other words, this sentence contains some underlying assumptions that are not explicitly stated and that I am missing out on.

Reply:
Will check on this with the dev team. @iam-veeramalla please suggest

Reply:
Argo CD CLI tool is a tool that is used to configure Argo CD through command line. Red Hat OpenShift GitOps does not support this binary. We instead recommend the users to use our OpenShift Console to achieve the required configuration.